redis/redis.go

```diff
@@ -13,11 +14,11 @@ type Pool interface {
 }
 
 type Conn interface {
-	Get(name string) (string, error)
-	Set(name string, value string) (bool, error)
-	SetNX(name string, value string, expiry time.Duration) (bool, error)
-	Eval(script *Script, keysAndArgs ...interface{}) (interface{}, error)
-	PTTL(name string) (time.Duration, error)
+	Get(ctx context.Context, name string) (string, error)
```

Comment:
While you're adding `context.Context` to `Conn` methods could you also add it to `Pool.Get()`? Redigo's `redis.Pool` also has `GetContext(ctx context.Context) (Conn, error)` which would be nice to use.

```
type Pool interface {
	Get(ctx context.Context) Conn
}
```

Reply:
I've added this in #56

Reply:
Good idea, more refinement in #57.